examples/dtls-echo/Makefile

```diff
@@ -24,52 +23,55 @@ BOARD_INSUFFICIENT_MEMORY := airfy-beacon calliope-mini cc2650stk maple-mini \
 # NOTE: 6LoWPAN will be included if IEEE802.15.4 devices are present
 USEMODULE += gnrc_netdev_default
 USEMODULE += auto_init_gnrc_netif
-# Specify the mandatory networking modules for IPv6 and sUDP
+# Specify the mandatory networking modules for IPv6 and UDP
 USEMODULE += gnrc_ipv6_router_default
 USEMODULE += gnrc_udp
-# Add a routing protocol
+USEMODULE += gnrc_sock_udp
+# Add a routing protocol (optional)
 USEMODULE += gnrc_rpl
-# This application dumps received packets to STDIO using the pktdump module
-USEMODULE += gnrc_pktdump
+
 # Additional networking modules that can be dropped if not needed
 USEMODULE += gnrc_icmpv6_echo
 # Add also the shell, some shell commands
 USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += ps
 
-#TinyDTLs (crypto.c) made use of pthread
- ifneq ($(BOARD),native)
- USEMODULE += pthread
- endif
 
-# Comment this out to disable code in RIOT that does safety checking
-# which is not needed in a production environment but helps in the
-# development process:
-CFLAGS += -DDEVELHELP
+# Uncomment to redefine port. Default: DTLS port (20220). Useful for FEBx range
+#DTLS_PORT = 20220
+#CFLAGS += -DDTLS_DEFAULT_PORT=$(DTLS_PORT)
 
-# NOTE: Add the package for TinyDTLS
-USEPKG += tinydtls
+# If enabled, DTLS client will spoofed a CoAP message (GET /riot/value)
```

Comment:
Just to be sure the following change will be enough clear: 
`DTLS-client will generate a "CoAP get /riot/value"  message instead of sending the user input`